elementary/files
# Focus rectangle - adjust padding; fix icon clearance & occasional truncation.

Fixes #82 
* Reduce number of variables used by TextRenderer, included deprecated.
* Fix some truncation problems in ListView for long files names
* Fix clearance between focus rectangle and icon for tall or wide icons.
* Reduce padding between text and focus rectangle.
* Remove variable text size code - it has no effect under default conditions.

TO TEST
Verify that there is no truncation or misposition of focus rectangle and text for all views, zoom-levels and name lengths in both LTR and RTL environments.

Verify that name editing as unaffected.  NOTE: There is a pre-existing bug for renaming in List View in RTL environment which is not fixed in this branch.
